packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -231,11 +235,38 @@ function MainContent({
           height='15rem'
           width='100%'
         >
-          <BarChart
-            data={stats?.data}
-            dateRange={selectedDateRange}
-            type={activeTab === 0 ? 'count' : 'session_time'}
-          />
+          {paramsValidationMessage ? (
+            <Box
+              align='center'
+              fill
+              justify='center'
+              pad='medium'
+            >
+              <SpacedText uppercase={false}>{paramsValidationMessage}</SpacedText>
+            </Box>
+          ) : noStats ? (
+            <Box
+              align='center'
+              fill
+              justify='center'
+              pad='medium'
+            >
+              <SpacedText uppercase={false}>No data found.</SpacedText>
+              <Text>
+                Start by{' '}
+                <Anchor href='https://www.zooniverse.org/projects'>
+                  classifying a project
+                </Anchor>
+                {' ' }now.
+              </Text>
```

Comment:
I think this `noStats` message should also include a suggestion like "Start by classifying a project now, or change the date range." It's likely volunteers visiting their stats page for the first time (and have not submitted a classification in the past 7 days) will see a 'no data' message before exploring the rest of the page. We should guide them to use the date range select.

Reply:
Yes, great suggestion!